owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -96,7 +96,7 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
         super.onCreate(savedInstanceState)
 
         // Protection against screen recording
-        if (!MainApp.isDeveloper) {
+        if (!MainApp.enabledLogging) {
```

Comment:
I think that this is not the same. In this case, we should not disable the protection when logs are enabled, am I right @jesmrec ?

Reply:
You are right. Screen protection should be active only in case it is a release version. For developer/debug/etc... it does not.